config/rmt.yml

```diff
@@ -12,5 +12,5 @@ http_client:
   proxy_password:
 
 scc:
-  username:
-  password:
+  username: <%= ENV['RMT_ORGANIZATION_USERNAME'] %>
+  password: <%= ENV['RMT_ORGANIZATION_PASSWORD'] %>
```

Comment:
Hmm, I'm not sure if we want that for the release RPM package.

Reply:
It's not usable without editing anyway, so I don't see any sense at keeping it old way either

Reply:
OK! :+1: